client/src/components/Accordion/AccordionBlock.js

```diff
@@ -17,7 +17,7 @@ const AccordionBlock = (props) => {
     <div className="accordion__block">
       <a
         className="accordion__title"
-        data-toggle="collapse"
+        data-bs-toggle="collapse"
```

Comment:
https://getbootstrap.com/docs/5.0/migration/#javascript
> Data attributes for all JavaScript plugins are now namespaced to help distinguish Bootstrap functionality from third parties and your own code. For example, we use `data-bs-toggle` instead of `data-toggle`.

Other `data-*` attributes that need changing _seem_ to only need to be changed if `data-bs-toggle` is also present - see [this third-party script](https://github.com/coliff/bootstrap-5-migrate-tool/blob/main/gulpfile.js) which seems to list all the relevant attributes (I didn't run this, just used it as a guide)